docs/docs/faq.mdx

```diff
@@ -154,6 +154,10 @@ When you create an SDK connection, it is linked to a specific environment. You c
 
 Throughout the GrowthBook application, we randomly celebrate key milestones like launching experiments with on-screen confetti. If you'd like to disable this, you can click on your avatar in the top right corner and select "Edit Profile". From there, you can disable the toggle for "Allow Celebrations". Please note this is persisted in your browser's local storage, so if you clear your browser's local storage, you will need to disable this again.
 
+## How can I tell which version of GrowthBook I'm running?
+
+Sometimes for debugging we'll ask which version of GrowthBook you're using. When you're logged in to GrowthBook, you can head to [app.growthbook.io](https://app.growthbook.io) and check the lower left corner to see "Build:" and the date of your version.
```

Comment:
Finding the version is not relevant for app.growthbook.io (Cloud).  It's only for self-hosted